src/components/Checkbox/Checkbox.tsx

```diff
@@ -32,16 +46,21 @@ export const Checkbox = forwardRef<HTMLButtonElement, CheckboxProps>(
         gap={1.5}
         position="relative"
         cursor={disabled ? "not-allowed" : "pointer"}
+        {...props}
       >
         <RadixCheckbox
           ref={ref}
           className={classNames(
             commonCheckbox,
-            error ? errorCheckbox : defaultCheckbox
+            error ? errorCheckbox : defaultCheckbox,
+            className
```

Comment:
**issue**: className placed here will be confusing, when I pass that prop i would expect to be attached to the wrapper not to some component down the tree

Reply:
Okay I will delete it